### PR TITLE
add condition

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15 @ReichEdvin @iulianatodoran

**Review by gomboskriszta15 — APPROVED** [review]

File: Assets/AltTester/Editor/AltTesterEditorWindow.cs [hunk 1]
```diff
@@ -214,19 +214,21 @@ public static void CheckDesktopVersionRequest()
                     {
                         regex = new System.Text.RegularExpressions.Regex(@"https://altom.com/app/uploads/AltTester/desktop/AltTesterDesktop[\w\.]*.dmg");
```
Comment (alexrotaru) [inline Assets/AltTester/Editor/AltTesterEditorWindow.cs:215]: this suggestion is not related to the bug, but it's a nice change, if you ask me :)
use a variable for the builds location, and use that variable here. if in the future we'll change to a new domain, it'll be easier to change it once.

**Comment by SandyEma9** [comment]
@alexrotaru I've resolve your comment

File: Assets/AltTester/Editor/AltTesterEditorWindow.cs [hunk 2]
```diff
@@ -62,6 +62,9 @@ public class AltTesterEditorWindow : UnityEditor.EditorWindow
         public static UnityEngine.Texture2D PortForwardingTexture;
         public static UnityEngine.Texture2D selectedTestsCountTexture;
 
+        private const string regexForWindows = @"https://altom.com/app/uploads/AltTester/desktop/AltTesterDesktop[\w\.]*.exe";
+        private const string regexForMacOS = @"https://altom.com/app/uploads/AltTester/desktop/AltTesterDesktop[\w\.]*.dmg";
+
         private static string downloadURL;
```
Comment (alexrotaru) [inline Assets/AltTester/Editor/AltTesterEditorWindow.cs:68]: the location path is the same. we don't need two variables for that. the only thing that differs is the extension of the file, which can be treated later, when you know the OS (you can add the extension after the regexp). this way, if you need to change the location, you would change it only once.

**Review by alexrotaru — APPROVED** [review]

**Pull request merged**